packages/http-client/src/document.ts

```diff
@@ -27,19 +27,9 @@ class Document extends Doctype {
     return new DocID(this.state.doctype, this.state.log[0])
   }
 
-  static async create (apiUrl: string, doctype: string, params: DocParams, context: Context, opts: DocOpts = {}): Promise<Document> {
```

Comment:
This whole function was dead code, best as I can tell.   I don't even think the `/create` endpoint in the daemon would be able to handle arguments passed the way they are being done here.